### DISCUSSION TITLE
Dose we have a plan to support global illumination?

### DISCUSSION THREAD

**Discussion by luciddream-tsin** [rokups/rbfx#406] [Q&A]
As the title.

**Comment by eugeneko** [comment]
Wow, someone used discussions!

Answer: perhaps?

If you describe what kind of GI you expect, I would give more specific answer.
Are we talking about mobile-friendly baked GI, realtime approximated voxel-based GIs, physically correct RTX-based GI?..

**Reply by luciddream-tsin** [reply]
Thanks for reply! 

Sorry for the not rigorous of my express. I'm a novice in compute graphics -_-
 
I think what I want to discuss is the second, real-time dynamic global illumination, but not based on hardware.

**Reply by eugeneko** [reply]
I definetely want to add some voxel-based relatime GI, but it is quite deep in the backlog of "things to do".

At the very least I will need to cleanup low-/mid-level renderer and migrate to 3rdparty library for API support (I am looking at Diligent renderer now). The good news is that this cleanup itself is quite high in backlog. Who knows, maybe I'll give realtime GI a try afterwards?

**Reply by luciddream-tsin** [reply]
Thank you very much for your work. 

I will also try to participate in the core of the project. 
Now I will try to comtribute some thing for simple work first.
